src/status_im/ui2/screens/chat/actions.cljs

```diff
@@ -0,0 +1,97 @@
+(ns status-im.ui2.screens.chat.actions
+  (:require
+   [status-im.chat.models :as chat.models]
+   [status-im.chat.models.pin-message :as models.pin-message]
+   [status-im.i18n.i18n :as i18n]
+   [status-im.constants :as constants]
+   [status-im.utils.handlers :refer [<sub >evt]]
+   [quo2.components.drawers.action-drawers :as drawer]))
+
+(defn hide-sheet-and-dispatch [event]
+  (>evt [:bottom-sheet/hide])
+  (>evt event))
+
+(defn show-profile-action [chat-id]
+  (hide-sheet-and-dispatch  [:chat.ui/show-profile chat-id])
+  (>evt [::models.pin-message/load-pin-messages chat-id]))
+
+(defn mark-all-read-action [chat-id]
+  (hide-sheet-and-dispatch  [:chat/mark-all-as-read chat-id]))
+
+(defn edit-nickname-action [chat-id]
+  (hide-sheet-and-dispatch  [:chat.ui/edit-nickname chat-id]))
+
+(defn mute-chat-action [chat-id]
+  (hide-sheet-and-dispatch  [::chat.models/mute-chat-toggled chat-id true]))
+
+(defn unmute-chat-action [chat-id]
+  (hide-sheet-and-dispatch  [::chat.models/mute-chat-toggled chat-id false]))
+
+(defn clear-history-action [chat-id]
+  (hide-sheet-and-dispatch [:chat.ui/clear-history-pressed chat-id]))
+
+(defn delete-chat-action [chat-id]
+  (hide-sheet-and-dispatch [:chat.ui/remove-chat-pressed chat-id]))
+
+(defn mute-chat-entry [muted chat-id]
+  {:icon :main-icons2/muted
+   :label (i18n/label
+           (if muted
+             :unmute-chat
+             :mute-chat))
+   :on-press (if muted
+               #(unmute-chat-action chat-id)
+               #(mute-chat-action chat-id))})
```

Comment:
Since this map is repeated, Is it better to abstract it into a function that takes `label`, `icon`, `danger` and `on-press` as params?